xbmc/cores/VideoPlayer/Process/wayland/ProcessInfoWayland.h

```diff
@@ -1,7 +1,5 @@
-#pragma once
```

Comment:
The `Keymap.h` removal is a separate commit in fact. Git just detects this as move when producing the diff for the whole history. There's nothing you can do about it as far as I know.